src/components/application_manager/src/commands/mobile/change_registration_request.cc

```diff
@@ -430,53 +436,48 @@ mobile_apis::Result::eType ChangeRegistrationRequest::CheckCoincidence() {
         return mobile_apis::Result::DUPLICATE_NAME;
       }
     }  // end vr check
-  }    // application for end
+  }  // application for end
   return mobile_apis::Result::SUCCESS;
 }
 
 bool ChangeRegistrationRequest::IsNicknameAllowed(
     const custom_str::CustomString& app_name) const {
   LOG4CXX_AUTO_TRACE(logger_);
-  ApplicationSharedPtr app =
-      application_manager::ApplicationManagerImpl::instance()->application(
-          connection_key());
+  ApplicationSharedPtr app  =
+      application_manager_.
+      application(connection_key());
 
   if (!app) {
-    LOG4CXX_ERROR(logger_,
-                  "Can't find appication with connection key "
-                      << connection_key());
+    LOG4CXX_ERROR(logger_, "Can't find appication with connection key "
+                  << connection_key());
     return false;
   }
 
-  const std::string policy_app_id = app->mobile_app_id();
```

Comment:
@LuxoftAKutsan , it`s applicable because this variable and getter applied to all places in GENIVI.